samples/graph-meeting-notification/csharp/README.md

```diff
@@ -0,0 +1,111 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates use of online meeting subscription and sends you the notifications in chat using bot.
```

Comment:
This is a sample application which demonstrates the use of online meeting subscription and sends you the notifications in chat using bot.

Reply:
Done